python/feathub/feature_views/feature_view.py

```diff
@@ -169,3 +172,30 @@ def _get_keys(self) -> Optional[List[str]]:
                 key_fields.extend(keys)
 
         return list(OrderedDict.fromkeys(key_fields))
+
+    def check_bounded(self, registry: Registry) -> bool:
+        return all(
+            [
+                cast(TableDescriptor, self.source).check_bounded(registry),
+                *[
+                    feature.transform.force_table_bounded
+                    or registry.get_features(
+                        feature.transform.table_name
+                    ).check_bounded(registry)
+                    for feature in self.get_resolved_features()
+                    if isinstance(feature.transform, JoinTransform)
```

Comment:
For join transform, suppose source is bounded and dimension table is unbounded (e.g. Kafka Stream), the output should still be bounded, right?

If yes, should we remove `force_table_bounded` from `JoinTransform`?